styles/common.css

```
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap');
+
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:opsz,wght@9..40,300;9..40,400;9..40,500;9..40,700&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+}
+
+.container {
+    width: 1000vh;
+    max-width: 1204px;
+}
+
+.navigation {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 230px;
+    font-family: DM Sans;
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 26px;
+}
+
+.logo-wrap img {
```

Já que a gente já tá mexendo nesse css acho melhor já fazer umas melhorias.
Por exemplo é uma boa prática não referenciar as tags do HTML diretamente, nesse caso é melhor crias uma classe pra essa imagem no HTML e usar a classe aqui.

Ex:
```suggestion
.logo-wrap .logo-img {
```

---

Feito!